Formula/frps.rb

```diff
@@ -0,0 +1,73 @@
+class Frps < Formula
+  desc "This is the server of frp (https://github.com/fatedier/frp)"
+  homepage "https://github.com/fatedier/frp"
+  url "https://github.com/fatedier/frp.git",
+      :tag      => "v0.24.1",
+      :revision => "cbf9c731a096ce33ec282ed913a25cb2e5f05303"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_OS"] = "darwin"
```

Comment:
Are these really needed? Because that'll exclude usage in linuxbrew.

Reply:
remove useless ENV[XC_OS] = darwin  on commit 7d6d68d